addons/website/views/snippets/s_references_grid.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_references_grid" name="References Grid">
+    <section class="s_references_grid o_cc o_cc1 pt64 pb64">
+        <div class="container">
+            <div class="row o_grid o_grid_mode" data-row-count="4">
+                <div class="o_grid_item o_colored_level g-height-5 g-col-lg-5 col-lg-5" style="grid-area: 1 / 1 / 5 / 6; z-index: 1;">
+                    <h2>Trusted references</h2>
+                    <p class="lead">We are in good company.</p>
+                    <a t-att-href="cta_btn_href" class="btn btn-primary"><t t-esc="cta_btn_text">See our case studies</t></a>
```

Comment:
```suggestion
                    <a href="#" class="btn btn-primary">See our case studies</a>
```

`cta_btn_text` and `cta_btn_href` will make this button dynamic based on what the user selected in the configurator. Here I think we want it to stay always "case studies" related, right ? 

See: https://github.com/odoo/odoo/pull/71162